conda_smithy/lint_recipe.py

```diff
@@ -122,6 +122,20 @@ def lintify(meta, recipe_dir=None):
         lints.append('The recipe `license` should not include the word '
                      '"License".')
 
+    # 11: There should be one empty line at the end of the file.
+    if recipe_dir is not None and os.path.exists(meta_fname):
+        with open(meta_fname, 'r') as f:
+            lines = f.read().split('\n')
+
+        end_empty_lines_count = 0
+        for i, line in enumerate(reversed(lines)):
+            if line != '':
+                break
```

Comment:
Hehe, yeah, maybe there is a super pythonic and readable way of making that a one liner, but yeah, I dont know if  it is really better. Thanks for the example @MSeifert04 :-)